hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleAccount.java

```diff
@@ -506,6 +510,14 @@ public void setStakedId(long stakedId) {
 		state().setStakedNum(stakedId);
 	}
 
+	public boolean mayHavePendingReward() {
+		return getStakedId() < 0 && !isDeclinedReward();
+	}
+
+	public long getUsableStakedNodeIdUnsafe() {
```

Comment:
I don't think this is unsafe, maybe `getStakedNodeAddressBookId()` would be clearer.